internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -790,3 +791,62 @@ func (r *ISBServiceRolloutReconciler) ChildNeedsUpdating(ctx context.Context, fr
 
 	return !specsEqual || !labelsEqual || !annotationsEqual, nil
 }
+
+func (r *ISBServiceRolloutReconciler) getCurrentChildCount(rolloutObject ctlrcommon.RolloutObject) (int32, bool) {
+	isbsvcRollout := rolloutObject.(*apiv1.ISBServiceRollout)
+	if isbsvcRollout.Status.NameCount == nil {
+		return int32(0), false
+	} else {
+		return *isbsvcRollout.Status.NameCount, true
+	}
+}
```

Comment:
These functions moved from the `isbservicerollout_progressive.go` file (same for the other rollouts)